blog/2024-05-16-recharging-tranche-strategy-tutorial.md

```diff
@@ -0,0 +1,306 @@
+
+This is a great treasury management or token management strategy for believers in Flare.
+
+### Market conditions
+The strategy works well while the token price is initially ranging around $0.03 and I've got capital to deploy or tokens to liquidate as it diverges significantly. 
+
+### Network parameters
+First I go to Raindex pubstrats and fork the [tranche space strategy](https://github.com/rainlanguage/rain.dex.pubstrats/blob/main/src/tranche-space.rain).
+
+Then I enter the network details, in this case Flare. I label each parameter so these don't conflict with my app settings. 
+
+```
+networks: 
+  flare-tranche: 
+    rpc: https://rpc.ankr.com/flare 
+    chain-id: 14
+    network-id: 14
+    currency: FLR
+    
+subgraphs:
+  flare-tranche: https://subgraphs.h20liquidity.tech/subgraphs/name/flare-0xb06202aA
+  
+orderbooks:
+  flare-tranche:
+    address: 0xb06202aA3Fe7d85171fB7aA5f17011d17E63f382
+    network: flare-tranche
+    subgraph: flare-tranche
+
+deployers:
+  flare-tranche:
+    address: 0x550878091b2B1506069F61ae59e3A5484Bca9166
+    network: flare-tranche
+
+tokens:
+  flare-wflr:
+    network: flare-tranche
+    address: 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d
+  flare-eusdt:
+    network: flare-tranche
+    address: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
+```
+
+
+### Token Parameters
+Then I enter tokens I want to trade. In this case it's wFLR and eUSDT. I get the contract address either from the Raindex pubstrats or from [Flarescan](https://flarescan.com/). Note I can have a basket of tokens not just a pair.
+
+```
+tokens:
+  flare-wflr:
+    network: flare-tranche
+    address: 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d
+  flare-eusdt:
+    network: flare-tranche
+    address: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
+```
+
+### Orders & Vaults
+I put up my buy and sells with the relevant orderbook and my vaults. I can deposit and withdraw tokens into this vault after my order is deployed.
+
+```
+orders:
+  # vault-id generated with `openssl rand -hex 32`
+  flare-tranche-buy:
+    orderbook: flare-tranche
+    inputs:
+      - token: flare-wflr
+        vault-id: 0x562bd75e19e548420f9f3da43a7d7d67c6344580256b952c9214192c445d6043
+    outputs:
+      - token: flare-eusdt
+        vault-id: 0x562bd75e19e548420f9f3da43a7d7d67c6344580256b952c9214192c445d6043
+  flare-tranche-sell:
+    orderbook: flare-tranche   
+    inputs:
+      - token: flare-eusdt
+        vault-id: 0x562bd75e19e548420f9f3da43a7d7d67c6344580256b952c9214192c445d6043
+    outputs:
+      - token: flare-wflr
+        vault-id: 0x562bd75e19e548420f9f3da43a7d7d67c6344580256b952c9214192c445d6043
+```
+
+### Scenarios
+I set up the parameters that are consistent across both buy and sell scenarios. This is scriptable. These include:
+* Subparser details
+* How many seconds a tranche space lasts for
```

Comment:
"tranche space" is not explained

Reply:
Adding a link to the reference where these are explained in detail